torch/_inductor/compile_fx.py

```diff
@@ -962,6 +962,7 @@ def compile_fx(
         with config.patch(
             {
                 "cpp_wrapper": False,
+                "triton.multi_kernel": 0,  # don't enable multi-kernel if cpp_wrapper is used
```

Comment:
True?

Reply:
I checked this a bit further today. Looks like with cpp_wrapper flag is on, we do 2 passes codegen. In the second pass, we call `CudaKernelParamCache.get(kernel_name) ` to retrieve the meta data (block sizes etc) for the kernel in order to compute the grid size. However when multi-kernel is on, the CudaKernelParamCache lookup returns None for some kernels. 

I don't know the root cause yet, need look a bit more into this.